lib/diagnostic.rb

```diff
@@ -5,7 +5,11 @@ class Diagnostic
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    long_entry = ''
+    enum.foreach() do |one_line|
```

Comment:
The better answer here would have been to use a comparison method from Enumerable, like `max` or `max_by`.
e.g.
`enum.max { |a, b| a.length <=> b.length }`
`enum.max_by { |entry| entry.length }`
`enum.max_by(&:length)`